orders/serializers.py

```diff
@@ -17,13 +18,26 @@ class Meta:
         ]
 
 
+class DriverUserSerializer(serializers.ModelSerializer):
```

Comment:
this serializer was tested with a list view and it returns null except if it is a driver

Reply:
this will return null in the friver data not in the diver + order 
move the verification the the API View